docs/docs/develop/create.md

```diff
@@ -8,40 +8,32 @@ tags:
 
 {% band %}
 
+  ## PatternFly Element generator
+
   Before you begin, make sure you've followed the 
   [Prerequisites](/docs/develop/setup#prerequisites) in 
   [Setup](/docs/develop/setup).
 
-  <h3>PatternFly Element generator</h3>
-
-  Use the PatternFly Element generator to start the scaffolding process. From 
-  the root directory of the PatternFly Elements  repository, run the following 
-  command.
+  Then use the PatternFly Element generator to start the scaffolding process. 
+  
+  From the root directory of the PatternFly Elements  repository, run the following 
+  command:
 
   ```bash
   npm run new
   ```
 
-  The generator will ask you a series of questions to set up your project.
-
-  1.  Your element name:
-      - Your element's name should be lowercase and needs to contain at least 
-        one hyphen. For rules on naming custom elements, refer to the W3C 
-        [Custom Elements Working 
-        Draft](https://www.w3.org/TR/custom-elements/#valid-custom-element-name).
-      - PatternFly Elements prefixes elements with `pf-`. However, prefix your 
-        elements with whatever fits your project.
-      - As an example, we'll create `pf-cool-element`.
+  The generator will then prompt you for the following:
 
-  1. The [NPM scope](https://docs.npmjs.com/cli/v6/using-npm/scope) for the 
-  package. For example, patternfly elements packages like `@patternfly/pf-core` 
-  are published under the `patternfly` scope.
+  * What is the element's tag name?
+    * Your element's name should be lowercase and needs to contain at least 
+      one hyphen. For rules on naming custom elements, refer to the W3C 
+      [Custom Elements Working Draft](https://www.w3.org/TR/custom-elements/#valid-custom-element-name).
```

Comment:
We should update this link to go to https://html.spec.whatwg.org/multipage/custom-elements.html#valid-custom-element-name I think, when going to the w3 link it has a popup linking to the github which links to this doc